docs/ethical-advertising.rst

```diff
@@ -0,0 +1,123 @@
+Ethical Advertising
+===================
+
+Read the Docs is a large free website on the internet.
+There is one proven business model to support this kind of site:
+Advertising.
+
+We talk a bit :ref:`below <ethical-info>` about our philosophy on advertising.
+
+If you're a marketer,
+we have :ref:`information <ethical-buy-ads>` on how to get in touch with us about running ads.
+
+.. _ethical-info:
+
+Our worldview
+-------------
+
+We're doing advertising properly:
+
+* We don't track users
+* We don't sell your data
+* We host everything ourselves, no third-party scripts
+
+What we're doing is newspaper advertising,
+on the internet.
+For a hundred years,
+newspapers put an ad on the page,
+some folks would see it,
+and advertisers would pay for this.
+This is our model.
+
+So much ad tech has been built to track users.
+You follow them across them web,
+from site to site,
+showing them the same ad.
+Then retailers sell your purchase data to try and attribute sales to advertising.
+Now there is an industry in doing `fake ad clicks`_ and other scams,
+which leads the ad industry to track you even more intrusively to know more about you.
+The current advertising industry is a vicious downward spiral.
+**We opt out.**
+
+As developers,
+we understand the `massive downsides`_ of the current advertising industry.
+From malware,
+slow site performance,
+and huge databases of your personal data sold to the highest bidder.
+**We opt out.**
+
+The trend in advertising is to have larger and larger ads.
+They should run before your content,
+they should take over the page,
+the bigger, weirder, or flashier the better.
+**We opt out.**
+
+We don't store personal information about you.
+We keep track of views and clicks.
+We don't build a profile of your personality to sell ads against.
+We only show high quality ads from companies that are of interest to developers.
+
+We are running a single,
+small,
+unobtrusive ad on documentation pages.
+The products should be interesting to you.
+They won't flash or move.
+
+We run the ads we want to have on our product,
+in a way that makes us feel good.
```

Comment:
`We run the ads we want to have on our product` is a weak way of saying `We display ads for products we believe in or want to support`

And `in a way that makes us feel good` is a overly broad way of saying `in a way that puts users over profit`